packages/terra-list/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* `isDisabled` prop added to fix accessibility issue to handle selectable and non-selectable items correctly.
```

Comment:
Should there be disabled styles applied to the list item when isDisabled = true?